### DISCUSSION TITLE
Translating Scratch Addons

### DISCUSSION THREAD

**Discussion by iqnite** [ScratchAddons/ScratchAddons#1044] [Ideas] [answered]
Hello, my Transifex username is philipp2007, as in Scratch, and I want to translate it into Italian and German.

**Comment by apple502j** [comment, marked as answer]
Done!

**Reply by iqnite** [reply]
How do I get to the Translate page?

**Reply by apple502j** [reply]
https://www.transifex.com/scratch-addons/scratch-addons-extension/dashboard/

**Reply by iqnite** [reply]
I can’t see Italian!

**Reply by WorldLanguages** [reply]
Added
